home/views.py

```diff
@@ -1,10 +1,11 @@
-from django.shortcuts import render
 from django.http import HttpResponse
+from django.shortcuts import render
 
 # Create your views here.
+from home.models import Student
 
 
-def home(request):
+def home(request):  # noqa
```

Comment:
Why noqa here?

Reply:
because flake8 swears at an argument that is not used, so I'll decide to comment out